.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+__Issues are used only for bug reports and feature requests.__
+
+If you have troubles with JanusGraph installation, configuration and integrations or you have questions about JanusGraph usage use this group to ask your questions:  
+https://groups.google.com/forum/#!forum/janusgraph-users
+
+If you have questions about Gremlin language usage use this group to ask your questions:  
```

Comment:
Again, a comma after _usage_ and I think it should be: _about **the** Gremlin language_, maybe even: _the Gremlin graph traversal language in general, use [...]_